src/styles/breakpoints.js

```diff
@@ -43,5 +51,22 @@ export default function createBreakpoints(
     return breakpoints[name];
   }
 
-  return { keys, values, up, down, only, getWidth };
+  let matched = priority
+    .reduce((a, k) => {
+      const media = window.matchMedia(breakpoints[k]);
```

Comment:
Well on the flip side,  If we do use media queries for all, the number of classes will go up significantly. I am not sure the performance consequence of JSS rendering so many classes in the `style` tag. If the performance is negligible This can be done.